kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -412,6 +412,11 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     context:
       "A coach can view learner progress in Kolibri.\n\nFor example, in the Coach > Reports section under the 'Progress' column they can see how many learners have started a lesson, or if a learner needs help.\n",
   },
+  questionsLeftLabel: {
+    message: '{remaining, number} questions left',
+    context:
+      "Indicates how many questions a learner has left to answer in a quiz. For example, it will show something like '5 questions left' or '1 question left' when a learner is taking a quiz.",
+  },
```

Comment:
Ah - one blocking note here. I missed this was part of the diff in my initial review and thought it was pre-existing when I saw this.

We'll need to use ICU syntax for the word "questions" here which allows translators to translate the various ways of expressing an amount of something. A similar string is defined [here in the `QuizCard` component](https://github.com/learningequality/kolibri/blob/ec915139ef840970e56d287d4dafc88937eda190/kolibri/plugins/learn/assets/src/views/cards/QuizCard/index.vue#L90-L94) which demonstrates how this works where we can say "when the value of `questionsLeft` is `one` then the word is `question` otherwise, use `questions`".

You could probably just copy that string directly into the `ProgressBar` component - note how you'll add a `$tr: {}` object to that component, then add the string inside of that object there.

Reply:
Interesting! ✅